cli/tests/e2e/e2e.test.ts

```diff
@@ -68,20 +80,86 @@ describe("e2e tests", function test() {
 
   const subsidyEnabled = false;
 
+  const setVerifyingKeysArgs: SetVerifyingKeysArgs = {
+    quiet: true,
+    stateTreeDepth: STATE_TREE_DEPTH,
+    intStateTreeDepth: INT_STATE_TREE_DEPTH,
+    messageTreeDepth: MSG_TREE_DEPTH,
+    voteOptionTreeDepth: VOTE_OPTION_TREE_DEPTH,
+    messageBatchDepth: MSG_BATCH_DEPTH,
+    processMessagesZkeyPath: processMessageTestZkeyPath,
+    tallyVotesZkeyPath: tallyVotesTestZkeyPath,
+  };
```

Comment:
Can we reuse these args for all the e2e tests? I noticed that almost the same args were using for e2e subsidy.

Reply:
I moved everything but genProof args inside constants